guides/source/routing.md

```diff
@@ -391,19 +405,20 @@ The articles resource here will have the following routes generated for it:
 | PATCH/PUT | /articles/:id(.:format)                      | articles#update   | article_path             |
 | DELETE    | /articles/:id(.:format)                      | articles#destroy  | article_path             |
 
-The [`shallow`][] method of the DSL creates a scope inside of which every nesting is shallow. This generates the same routes as the previous example:
+The [`shallow`][] method with a block creates a scope inside of which every nesting is shallow. This generates the same routes as the previous example:
 
 ```ruby
 shallow do
   resources :articles do
     resources :comments
     resources :quotes
-    resources :drafts
   end
 end
 ```
 
-There exist two options for `scope` to customize shallow routes. `:shallow_path` prefixes member paths with the specified parameter:
+There are two options that can be used with `scope` to customize shallow routes - `:shallow_path` and `:shallow_prefix`.
```

Comment:
Maybe it would be useful to mention why we'd use `shallow_path` and `shallow_prefix`.

Reply:
Other than to customize it, as it say, no other reason I can think of off hand.